Userland/Libraries/LibWasm/AbstractMachine/BytecodeInterpreter.cpp

```diff
@@ -159,6 +159,47 @@ void BytecodeInterpreter::load_and_push_mxn(Configuration& configuration, Instru
     configuration.stack().peek() = Value(bit_cast<u128>(convert_vector<V128>(bytes)));
 }
 
+template<size_t N>
+void BytecodeInterpreter::load_and_push_lane_n(Configuration& configuration, Instruction const& instruction)
+{
+    auto memarg_and_lane = instruction.arguments().get<Instruction::MemoryAndLaneArgument>();
+    auto& address = configuration.frame().module().memories()[memarg_and_lane.memory.memory_index.value()];
+    auto memory = configuration.store().get(address);
+    auto vector = *configuration.stack().pop().get<Value>().to<u128>();
+    auto base = *configuration.stack().pop().get<Value>().to<u32>();
+    u64 instance_address = static_cast<u64>(bit_cast<u32>(base)) + memarg_and_lane.memory.offset;
+    Checked addition { instance_address };
+    addition += N / 8;
+    if (addition.has_overflow() || addition.value() > memory->size()) {
+        m_trap = Trap { "Memory access out of bounds" };
+        return;
+    }
+    auto slice = memory->data().bytes().slice(instance_address, N / 8);
+    auto dst = reinterpret_cast<u8*>(&vector) + memarg_and_lane.lane * N / 8;
```

Comment:
_please_ don't use `reinterpret_cast` on pointers.
Almost every meaningful use of this is better expressed as `static_cast` or `bit_cast`. in this case `static_cast` is fine as you're not casting to a higher alignment requirement.

Reply:
Correction: static_cast isn't valid here, so just bit_cast instead pls.